handlers/validator.go

```diff
@@ -423,14 +423,19 @@ func Validator(w http.ResponseWriter, r *http.Request) {
 			}
 
 			timeToWithdrawal := utils.GetTimeToNextWithdrawal(distance)
+			address, err := utils.WithdrawalCredentialsToAddress(validatorPageData.WithdrawCredentials)
+			if err != nil {
+				logger.WithError(err).Error("error invalid withdrawal credentials")
```

Comment:
I have talked about this with @recy21 and the conclusion is that this should be a warning.

Furthermore, `WithdrawalCredentialsToAddress` does not return an address in this case. The handling below maybe should be updated to show something like "Invalid Address" or "N/A"?
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/5203a6bb9f78b97f65312d3b0e071855dba7978e/handlers/validator.go#L438

Reply:
As discussed if there is no valid address we show N/A for the withdrawal credentials in the table and don't link anywhere.

If you want to test this I used
http://localhost:8081/validator/0xadc3f47051a6fb515636121b10b201d2cfb44f9dc5c493402eefbc198fcdf472cc7f55eed21643ac278b1bf9686d4d24#withdrawals
and forced
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/67a363939653acb1ed60c1dfa08ac5bf74c4e666/handlers/validator.go#L393
to true.